tests/csv_46.t

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+. bash_tap_fd.sh
+
+PASSWD=$(get_password)
+CMD="$(get_script_python3) --format csv"
+TEST="$(get_test_data)/test_profile_firefox_46/"
+
+${bashtap_org_pwd}/diff_csv.py <(echo ${PASSWD} | ${CMD} ${TEST} | grep doesntexist || kill $$) <(get_user_data "doesntexist_csv")
```

Comment:
Feel free to use the same 'dumb' approach from before of using `diff -u`. No need to check if the output is parseable since other users may not be using something else to read the csv format.
Bruteforcing on the output being exactly what is in the expected file should be sufficient.